cc.arduino.arduinoide2.json

```diff
@@ -0,0 +1,116 @@
+{
+    "app-id": "cc.arduino.arduinoide2",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "arduino-ide",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
does it really support wayland?

Reply:
i believe the devs are aiming to support wayland (https://github.com/arduino/arduino-ide/issues/704) and the application does work (in my testing) on wayland.

Reply:
with these permissions it could just use Xwayland (X11). `fallback-x11` should be used when the application support wayland. Which would cause a failure if the application doesn't support wayland.

Reply:
for now ill remove the wayland then.